### PR TITLE
Fix bug preventing all data being shown

### LINKED ISSUE #2: All data missing on completing the final pub
Currently there is no special case for when there is not a next pub (i.e. when all pubs are completed). This causes an error that means no data is displayed in the interface.

### PR DESCRIPTION
**Pull request by KMayne**
Previously nextPub's members were being accessed when there was no next
pub (i.e. at the end of the crawl). This meant that an exception occured
in the web app, preventing the data from being rendered correctly.

Fixes #2

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -4,6 +4,5 @@ root = true
 charset = utf-8
 indent_style = space
 indent_size = 2
-end_of_line = lf
```
Comment (ThomasBower) [inline .editorconfig]: Does this need to be committed?

File: src/components/RoutePage.vue [hunk 2]
```diff
@@ -10,6 +10,9 @@
       <i class="material-icons">done</i>
     </button>
   </section>
+  <section v-else>
+    <h1>Congratulations! You have completed all the pubs 🍻</h1>
```
Comment (ThomasBower) [inline src/components/RoutePage.vue]: Nit: "You have completed all the pubs" reads a little bit weirdly to me. Maybe something like "You have completed the pub crawl!"? Not urgent.

File: .editorconfig (hunk 1 above)
Comment (KMayne) [reply .editorconfig]: Yes

File: src/components/RoutePage.vue (hunk 2 above)
Comment (KMayne) [reply src/components/RoutePage.vue]: Agreed 👍

**Pull request merged**